Documentation/04_to_2_snapshot_migration.md

```diff
@@ -0,0 +1,31 @@
+## Snapshot Migration
+
+You can migrate a snapshot of your data from a v0.4.9+ cluster into a new etcd 2.2 cluster using a snapshot migration. After snapshot migration, the etcd indexes of your data will change.
```

Comment:
Something like this: Many etcd applications rely on these indexes to behave correctly so this operation should only be done while all applications are stopped.